projects/uswds-formly/src/lib/types/input.ts

```diff
@@ -11,12 +12,28 @@ import { FieldType } from '@ngx-formly/core';
       }"
       class="usa-input"
       [placeholder]="to.placeholder"
+      [usaCharacterCount]="to.characterCount"
       [name]="to.name ? to.name : 'input'"
       [formlyAttributes]="field"
       [type]="to.inputType ? to.inputType : 'text'"
       [formControl]="formControl"
     />
-
+    </ng-container>
+    <ng-container *ngIf="!to.characterCount">
```

Comment:
Is it necessary to create two separate templates based on character count? looking at the usaCharacterCount directive, it seems to do nothing if a characterCount number is not given.